packages/office-ui-fabric-react/src/components/Pivot/PivotItem.tsx

```diff
@@ -3,6 +3,14 @@ import { BaseComponent, getNativeProps, divProperties } from '../../Utilities';
 import { IPivotItemProps } from './PivotItem.types';
 
 export class PivotItem extends BaseComponent<IPivotItemProps, {}> {
+  constructor(props: IPivotItemProps) {
+    super(props);
+
+    this._warnDeprecations({
+      linkText: 'headerText'
+    });
```

Comment:
Missing `initialSelectedKey`?

Reply:
No, that's a prop on Pivot itself, not PivotItem

Reply:
:+1: